### DISCUSSION TITLE
Fluent Ribbon Statusbar usage

### DISCUSSION THREAD

**Discussion by Rickyd56** [fluentribbon/Fluent.Ribbon#1209] [Q&A] [answered]
I am working with this great ribbon tool, but cannot figure out where to place the Statusbar code.  I read the documentation, but it didn't provide where the xml code goes within the Mainwindow xml.  Could you provide help or an example in an application?  Thanks

**Comment by batzen** [comment, marked as answer]
The showcase application contains almost every aspect of Fluent.Ribbon.

The `StatusBar` is used at https://github.com/fluentribbon/Fluent.Ribbon/blob/643257718643559779bb40219d522841e8e3a0f0/Fluent.Ribbon.Showcase/TestContent.xaml#L3770 for example.

**Reply by Rickyd56** [reply]
Thank you for the quick response.  I have successfully used the statusbar control and it is great.

**Reply by batzen** [reply]
Glad i could help.
